--- origsrc/libsoup-2.4.1/configure.in	2008-04-07 08:23:56.000000000 -0500
+++ src/libsoup-2.4.1/configure.in	2008-05-28 01:02:03.468750000 -0500
@@ -234,6 +234,10 @@
 			APACHE_PHP_MODULE_DIR="$dir"
 			APACHE_PHP_MODULE=libphp5.so
 		    fi
+		    if test -f $dir/cygphp5.so; then
+			APACHE_PHP_MODULE_DIR="$dir"
+			APACHE_PHP_MODULE=cygphp5.so
+		    fi
 		done])
     AC_MSG_RESULT($APACHE_MODULE_DIR)
     AC_SUBST(APACHE_MODULE_DIR)
